Ports/cmake/package.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=cmake
+version=3.19.4
+useconfigure=false
+files="https://github.com/Kitware/CMake/releases/download/v$version/cmake-$version.tar.gz cmake-$version.tar.gz"
+depends=""
```

Comment:
Please add to AvailablePorts.md :)

Reply:
You keep telling me this, and I keep forgetting about it ^^'

Reply:
Linting of available ports table when

Reply:
@BenWiederhake ^

Reply:
> Linting of available ports table when

https://www.nieuweplaat.nl/wp-content/uploads/2015/11/12045799_892741494130158_3452631501732855933_o.jpg